src/main/kotlin/sk/streetofcode/webapi/api/YouTubeService.kt

```diff
@@ -5,4 +5,5 @@ import com.google.api.services.youtube.model.Video
 interface YouTubeService {
     fun getVideos(): List<Video>
     fun getVideo(id: String): Video
+    fun clearCache()
```

Comment:
I would remove this. Also if something like this would be needed in future, shouldn't we gourd it under @admin privileges?

Reply:
I think it might be useful once in 10 years, costs nothing. I don't think it needs to be protected - we only care about the cache during build, so if someone clears it, nothing would really happen even if they did it during a build. Also if it were protected how would we call it? Would you copy your JWT token from browser and paste it into curl? The idea is that it should be simple to call, which you can now do simply via `curl --location 'http://localhost:8080/youtube/clear-cache'` and that's it.

Reply:
Ok